libs/api/domains/communications/src/lib/communications.service.ts

```diff
@@ -135,6 +135,16 @@ export class CommunicationsService {
       return false
     }
 
+    let recipient = form.recipient
+
+    const emailConfig = form.recipientFormFieldDecider?.emailConfig
+    const key = input.recipientFormFieldDeciderValue as string
```

Comment:
This might still become the key `"undefined"`, which might make debugging _very_ hard if for any reason `emailConfig` unintentionally has the `"undefined"` key.